src/main/kotlin/org/rust/ide/typing/paste/RsConvertJsonToStructCopyPasteProcessor.kt

```diff
@@ -142,8 +146,16 @@ private fun createAndInsertStruct(
     return inserted
 }
 
-fun generateStruct(factory: RsPsiFactory, struct: Struct, nameMap: Map<Struct, String>): RsStructItem? {
+fun generateStruct(
+    factory: RsPsiFactory,
+    struct: Struct,
+    nameMap: Map<Struct, String>,
+    hasSerdeDependency: Boolean
+): RsStructItem? {
     val structString = buildString {
+        if (hasSerdeDependency) {
+            append("#[derive(serde::Serialize, serde::Deserialize)]\n")
```

Comment:
Let's add imports and use `#[derive(Serialize, Deserialize)]`

Reply:
Changed.